meine_stadt_transparent/settings/__init__.py

```diff
@@ -53,6 +53,9 @@
         "MAILJET_SECRET_KEY": env.str("MAILJET_SECRET_KEY"),
     }
     EMAIL_BACKEND = "anymail.backends.mailjet.EmailBackend"
+else:
+    EMAIL_CONFIG = env.email_url('EMAIL_URL')
+    vars().update(EMAIL_CONFIG)
```

Comment:
We can omit the default value:

```suggestion
elif "EMAIL_URL" in env:
    # If EMAIL_URL is not configured, django's SMTP defaults will be used
    EMAIL_CONFIG = env.email_url('EMAIL_URL')
    vars().update(EMAIL_CONFIG)
```

Reply:
Changed in https://github.com/meine-stadt-transparent/meine-stadt-transparent/pull/332/commits/958ba0581a525ff2310ec57559385a3dbbbbbc44